tests/python/pants_test/pants_run_integration_test.py

```diff
@@ -274,7 +274,7 @@ def run_pants_with_workdir_without_waiting(self, command, workdir, config=None,
       env['PANTS_PROFILE'] = prof
       # Make a note the subprocess command, so the user can correctly interpret the profile files.
       with open('{}.cmd'.format(prof), 'w') as fp:
-        fp.write(b' '.join(pants_command))
+        fp.write(' '.join(pants_command))
```

Comment:
`pants_command` is in unicode and file opened in text_mode, so I figured it makes most sense to stay with text.